client/landing/stepper/declarative-flow/internals/hooks/use-step-navigation-with-tracking/index.ts

```diff
@@ -36,7 +37,8 @@ export const useStepNavigationWithTracking = ( {
 					intent,
 					flow.name,
 					currentStepRoute,
-					flow.variantSlug
+					flow.variantSlug,
+					flow.useTracksEventProps?.( STEPPER_TRACKS_EVENT_STEP_NAV_SUBMIT )
```

Comment:
This is throwing an error because we're calling a hook (`flow.useTracksEventProps`) inside a normal function (`submit`).

We should probably call `flow.useTracksEventProps` outside submit.

Reply:
Ouch yes indeed. Weirdly never flagged in my editor (old man's broken dependency i guess 😁 )

I'll move it to a callback or something

Reply:
I think you can just move it a few lines up.